en/contribute/git_examples.md

```diff
@@ -72,30 +82,122 @@ For a good commit message, please refer to [Contributing](../contribute/README.m
   Responsible members of PX4 will now have a look at your contribution and decide if they want to integrate it.
   Check if they have questions on your changes every once in a while.
 
+
+## Changing Source Trees
```

Comment:
Hi @beniaminopozzan 

OK, so this is new section on switching branches/tags. Not sure about the name.

This is referenced below in the fetching releases sections as you suggested, and they also use the commands to clean up and get submodules.